frontend/src/components/facilities/FacilityInfo.vue

```diff
@@ -170,6 +170,78 @@
             </v-row>
           </v-col>
         </v-row>
+
+        <v-divider />
```

Comment:
if a facility has no additional addresses, we'll have a redundant divider. can we put both divider and expansion-panels inside a div then add v-if to that div? Or can we remove that v-divider because it doesn't look much different if we have it or not?

Reply:
you are right good point - i'll fix this